src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -181,7 +182,7 @@ export default class EthereumBlockchainService implements BlockchainService {
     }
 
     const txReceipt: providers.TransactionReceipt = await this.wallet.provider.waitForTransaction(
-      txResponse.hash, 1, transactionTimeoutSecs * 1000);
+      txResponse.hash, NUM_BLOCKS_TO_WAIT, transactionTimeoutSecs * 1000);
```

Comment:
Will we need to increase the `transactionTimeoutSecs` proportionally?  Depends on how the ethers provider works and how it applies that timeout. Does the timeout just apply for waiting for it to be initially mined?  Or does it apply to the entire `waitForTransaction` operation including waiting for extra block confirmations?

Reply:
https://docs.ethers.io/v5/single-page/#/v5/api/providers/provider/-%23-Provider-waitForTransaction

Timeout is the frequency of queries. Confirmations is how many blocks need to be mined before the function returns.

https://github.com/ethers-io/ethers.js/blob/master/packages/providers/src.ts/base-provider.ts#L927

Reply:
> Timeout is the frequency of queries. 

I don't really know what that means.  But if I'm reading the code from ethers right it looks like the timeout applies to the entire `waitForTransaction` call, including waiting for additional block confirmations.  That means we probably need to increase the timeout we're setting here proportional to the number of additional blocks we're waiting on.

Reply:
to clarify, this probably isn't a code change, but rather a change in the environment variable we set for `transactionTimeoutSecs`.  Which may in turn have ripple effects on how frequently we can anchor, and on the timeouts we set in the integration tests

Reply:
Oh i though `Timeout` specified poll interval.

Reply:
Naw, that's [provider.pollingInterval](https://github.com/ceramicnetwork/ceramic-anchor-service/blob/cceb3e5cbf863b472d9fae4c33bca18239fd6ebd/src/services/blockchain/ethereum/ethereum-blockchain-service.ts#L44)